packages/@aws-cdk-testing/framework-integ/test/aws-ec2/test/integ.bastion-host-arm-support.js.snapshot/TestStack.template.json

```diff
@@ -18,9 +18,6 @@
   "VPCPublicSubnet1SubnetB4246D30": {
```

Comment:
I updated the old integration tests, since AL2023 is the new default with the feature flag on. I added an integration test disabling the flag, which can be removed when v3 is released.